packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -737,7 +741,28 @@ export class RemoteMediaManager extends EventsScope {
     });
 
     LoggerProxy.logger.log(
-      `RemoteMediaManager#updateVideoReceiveSlots --> receive slots updated: unused=${this.slots.video.unused.length}, activeSpeaker=${this.slots.video.activeSpeaker.length}, receiverSelected=${this.slots.video.receiverSelected.length}\n${logMessage}`
+      `RemoteMediaManager#logMainVideoReceiveSlots --> MAIN VIDEO receive slots: unused=${this.slots.video.unused.length}, activeSpeaker=${this.slots.video.activeSpeaker.length}, receiverSelected=${this.slots.video.receiverSelected.length}${logMessage}`
+    );
+  }
+
+  /** Logs all current receive slots */
+  public logAllReceiveSlots() {
+    LoggerProxy.logger.log(
```

Comment:
Shouldn't we define a series of other functions while we're in here, called 

`logMainAudioReceiveSlots` and `logSlidesVideoReceiveSlots` and `logSlidesAudioReceiveSlots` and have this function just sort of call each of those (would be nice if it could be done in parallel but oh well) 

that way we'd have some symmetry, and a singular point for each that could be used, as logMainVideoReceiveSlots has?

Reply:
done